frontend/src/Pages/AllUser/Register/Register.jsx

```
@@ -0,0 +1,89 @@
+        <h2 className="text-center text-3xl font-bold text-indigo-600">Sign Up</h2>
+        <form onSubmit={handleSignUp} className="space-y-4 mt-6">
+          <div>
+            <label className="block text-sm font-medium text-gray-700">Name</label>
+            <input type="text" className="mt-1 p-2 w-full border rounded-md" value={name} onChange={(e) => setName(e.target.value)} required />
+          </div>
+          <div>
+            <label className="block text-sm font-medium text-gray-700">Email</label>
+            <input type="email" className="mt-1 p-2 w-full border rounded-md" value={email} onChange={(e) => setEmail(e.target.value)} required />
+          </div>
+          <div>
+            <label className="block text-sm font-medium text-gray-700">Password</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)